### PR TITLE
doc: update the release schedule for proposed 11

### PR DESCRIPTION
**Pull request by jasnell**
* Update the release schedule for the proposed 11.0.0 release date
* Reorg page a bit to group End-of-life releases.

### CODE REVIEW

**Comment by watson** [comment]
With the release of 10.0.0 yesterday, this PR is already out of date 😄

**Comment by jasnell** [comment]
Yeah, I was planning to update it after if it didn't land before :)

File: README.md [hunk 1]
```diff
@@ -4,15 +4,12 @@
...
-| 5.x     | **End-of-Life**     |            | 2015-10-29     |                  |                       | 2016-06-30                |
 | [6.x][] | **Active LTS**      | [Boron][]  | 2016-04-26     | 2016-10-18       | 2018-04-30            | April 2019                |
-| 7.x     | **End-of-Life**     |            | 2016-10-25     |                  |                       | 2017-06-30                |
 | [8.x][] | **Active LTS**      | [Carbon][] | 2017-05-30     | 2017-10-31       | April 2019            | December 2019<sup>1</sup> |
 | 9.x     | **Current Release** |            | 2017-10-01     |                  |                       | June 2018                 |
-| 10.x    | **Pending**         | Pending    | Apr 2018       | October 2018     | April 2020            | April 2021                |
+| 10.x    | **Pending**         | Dubnium    | 2018-04-24     | October 2018     | April 2020            | April 2021                |
+| 11.x    | **Pending**         |            | 2018-04-23     |                  |                       | June 2019                 |
```
Comment (targos) [inline README.md]: s/2018-04-23/2018-10-23/

**Comment by igor-savin-ht** [comment]
@alasdairhurst In either case, it's safe to mark it's current state as "End of Life" on the roadmap :)

File: README.md [hunk 2]
```diff
@@ -4,11 +4,10 @@
...
-| [4.x][] | **Maintenance LTS** | [Argon][]  | 2015-09-08     | 2015-10-01       | 2017-04-01            | 2018-04-30                |
 | [6.x][] | **Active LTS**      | [Boron][]  | 2016-04-26     | 2016-10-18       | 2018-04-30            | April 2019                |
 | [8.x][] | **Active LTS**      | [Carbon][] | 2017-05-30     | 2017-10-31       | April 2019            | December 2019<sup>1</sup> |
-| 9.x     | **Current Release** |            | 2017-10-01     |                  |                       | June 2018                 |
-| 10.x    | **Pending**         | Dubnium    | 2018-04-24     | October 2018     | April 2020            | April 2021                |
+| 9.x     |                     |            | 2017-10-01     |                  |                       | June 2018                 |
+| 10.x    | **Current Release** | Dubnium    | 2018-04-24     | October 2018     | April 2020            | April 2021                |
 | 11.x    | **Pending**         |            | 2018-04-23     |                  |                       | June 2019                 |
```
[the remark this answers is not in the document]
Comment (MylesBorins) [reply README.md]: what specifically is incorrect?

[the remark this answers is not in the document]
Comment (targos) [reply README.md]: s/2018-04-23/2018-10-23/

File: README.md [hunk 3]
```diff
@@ -4,15 +4,11 @@
...
-| [4.x][] | **Maintenance LTS** | [Argon][]  | 2015-09-08     | 2015-10-01       | 2017-04-01            | 2018-04-30                |
-| 5.x     | **End-of-Life**     |            | 2015-10-29     |                  |                       | 2016-06-30                |
 | [6.x][] | **Active LTS**      | [Boron][]  | 2016-04-26     | 2016-10-18       | 2018-04-30            | April 2019                |
-| 7.x     | **End-of-Life**     |            | 2016-10-25     |                  |                       | 2017-06-30                |
 | [8.x][] | **Active LTS**      | [Carbon][] | 2017-05-30     | 2017-10-31       | April 2019            | December 2019<sup>1</sup> |
-| 9.x     | **Current Release** |            | 2017-10-01     |                  |                       | June 2018                 |
-| 10.x    | **Pending**         | Pending    | Apr 2018       | October 2018     | April 2020            | April 2021                |
+| 9.x     |                     |            | 2017-10-01     |                  |                       | June 2018                 |
```
Comment (chicoxyzzy) [inline README.md]: Would it be better if status for Node 9 will be just **Active**?

**Comment by MylesBorins** [comment]
landed in de099b4...55bdd31

**Review by yvettep321 — COMMENTED** [review]
thank you

**Pull request closed without merging**